packages/ses/src/error/tame-v8-error-constructor.js

```diff
@@ -214,23 +222,41 @@ export const tameV8ErrorConstructor = (
     // string associated with an error.
     // See https://tc39.es/proposal-error-stacks/
     getStackString(error) {
+      let stackString = weakmapGet(stackStrings, error);
+
+      if (stackString !== undefined) {
+        return stackString;
+      }
+
       if (!weakmapHas(ssts, error)) {
         // eslint-disable-next-line no-void
         void error.stack;
       }
+      stackString = weakmapGet(stackStrings, error);
+      if (stackString !== undefined) {
```

Comment:
It would be true if `errorTaming === 'unsafe'`, in which case `prepareStackTrace` (effectively triggered by line 233), would save the stackString directly instead of the CallSite array.

It is actually necessary, but I can add a comment clarifying.